Control coverage/Feature/VirtualMachine.md

```diff
@@ -1429,24 +1411,19 @@ Known OS/framework vulnerabilities in a system can be easy targets for attackers
 
 ### Control Spec 
 
-> **Passed:** 
-> Passed condition
+> **Verify:** 
+> Virtual Machine's Operating System (OS) type is considered for evaluation.
```

Comment:
Should this be "is **not** considered for evaluation"?

Reply:
The code checks whether the VM shud be considered for evaluation. Hence, I believe it should remain as is.